Cython/Compiler/ExprNodes.py

```diff
@@ -196,9 +196,9 @@ def make_dedup_key(outer_type, item_nodes):
 
     @param outer_type: The type of the outer container.
     @param item_nodes: A sequence of constant nodes that will be traversed recursively.
-    @return: A tuple that can be used as a dict key for deduplication.
+    @return: A hashable, constant sequence that can be used as a dict key for deduplication.
```

Comment:
No, it actually returns a 2-tuple, two values: a type, and a tuple or (newly) frozenset.
```suggestion
    @return: A tuple that can be used as a dict key for deduplication.
```

Reply:
I added some detail here
```
    @return: A 2-element tuple that can be used as a dict key for deduplication.
                The first element is outer_type.
                The second element is a hashable, constant sequence
```

Please pardon my stubbornness. I spent a lot of time understanding this function, so I assume other new contributors could save some time from it